server/liveblog/themes/themes_assets/amp/template.html

```diff
@@ -0,0 +1,72 @@
+<!-- inject:template-content -->
+<div class="lb-timeline {{ settings.language }}">
+  {% if settings.showTitle and blog.title %}
+  <h1>{{ blog.title }}</h1>
+  {% endif %}
+
+  {% if settings.showDescription and blog.description %}
+  <p>{{ blog.description | safe }}</p>
+  {% endif %}
+
+  {% if settings.showImage and blog.picture_url %}
+  <amp-img src="{{ blog.picture_url }}"></amp-img>
```

Comment:
`amp-img` requires `height` and `width` from what I know,
That could be retrieved from the renditions, aswell `srcset` can be used here.